DOP-4909: feedback form works on mobile with new action bar

seungpark opened the pull request:
### Stories/Links:

DOP-4909

- Marking the destination branch as DOP-4897 as mobile behavior was updated on this branch
- Feedback card should take up the whole screen on mobile as per [figma design](https://www.figma.com/design/7cR6eWtFvKTnJIPL8ZsTaB/Content-Template-Projects?node-id=7488-6545&t=ZSe2b6fwRzBrQhC8-0)


### Staging Links:

[landing staged](https://docs-mongodb-org-stg.s3.us-east-2.amazonaws.com/master/landing/seung.park/DOP-4909/index.html)

### Notes:
- Did not stage other projects as it seems pretty standard on all pages. Feel free to request other staging links if needed

### README updates

- - [ ] This PR introduces changes that should be reflected in the README, and I have made those updates.
- - [X] This PR does not introduce changes that should be reflected in the README

rayangler commented on src/components/Widgets/FeedbackWidget/FeedbackCard.js:
```diff
@@ -16,9 +16,7 @@ import useNoScroll from './hooks/useNoScroll';
-    right: 0;
-    top: 0;
-    bottom: 60px;
+    height: 100%;
```
Looks like this height is causing the card container to extend well beyond the viewport (maybe because it's in a portal? not sure). This makes it so that scrolling further down the page and clicking the feedback menu results in a blank screen, with the actual feedback items further up

seungpark replied:
thanks for this catch! changing to fixed position

rayangler reviewed (commented):
Wanted to flag a couple of things:
* (Nit) Should the PR title say "from" or "form"?
* Opening the mobile side nav and then clicking on a rating causes it to disappear immediately. Seems like the form is still in the HTML but it's hidden due to the CSS. Is this something to be addressed in this PR or separately?

seungpark commented:
> Wanted to flag a couple of things:
> 
> * (Nit) Should the PR title say "from" or "form"?
> * Opening the mobile side nav and then clicking on a rating causes it to disappear immediately. Seems like the form is still in the HTML but it's hidden due to the CSS. Is this something to be addressed in this PR or separately?

Thanks for the behavioral catch. This was due to a [useClickOutside](https://github.com/mongodb/snooty/pull/1189/files#diff-a4d0df6952218a4f6afbca3fa05ba9db1228e7a4a03391d9aec21b40352a870dR14) previously added for closing feedback when user clicks outside of the desktop version. fixed and addressed! 

And updated the title :)

seungpark commented on src/components/Widgets/FeedbackWidget/FeedbackForm.js:
```diff
@@ -23,6 +23,14 @@ const formStyle = css`
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
```
for people with square windows! done
